pkg/security/resolvers/usersessions/resolver_linux.go

```diff
@@ -44,7 +44,7 @@ func (e *UserSessionData) UnmarshalBinary(data []byte) error {
 	}
 
 	e.SessionType = usersession.Type(data[0])
-	e.RawData += model.NullTerminatedString(data[1:])
+	e.RawData += model.NullTerminatedString(data[1:240])
```

Comment:
This is the main bug here, we were reading after the end of the buffer, in the padding